engine/src/python/infra/get_string.cpp

```diff
@@ -38,32 +37,36 @@ const std::string GetString(const std::string function_name,
                             const std::string module_name,
                             const std::string file_name,
                             PyObject* args) {
-    if(!boost::filesystem::exists(file_name)) {
```

Comment:
Why remove the check for the file? Is it because we no longer use absolute paths in the name?

Reply:
Yes. For that reason, and also because this function doesn't have enough information to know the full path to the file. As far as it's concerned, the Python file could be in one of a number of locations.

Reply:
ok